content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,254 @@
+
+The export script has 2 primary sections: the function [`export_standalone_project`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L19) and the [startup code that only runs if the script is invoked by the CLI](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L187). In depth discussion on these two sections will be made in the Developer Guide in the future.
+
+
+### Usage
+To use the export script, first ensure you have the necessary seedlist files for your project (as mentioned in the pre-requisites).
+
+You can issue the arguments for this script at the same time that you are running the `export-project` command, so long as you are using the script as your designated export script. The arguments specific to the script will be deferred until the script begins running.
+
+The arguments are as follows:
+| Argument Name | Description | Required? |
+| - | - | - |
+| [`--script-help`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L202) | Display the help information specifically for the export script. | no |
+| [`--output-path`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L203) | Defines the location on the system where the release layout directory for the project should be located. For each desired launcher for your project, a separate release folder will be prepared. | yes |
+| [`--config`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L204) | Defines the CMake build configuration when building the project's binaries, such as GameLauncher. Options are either `profile` or `release`. Default is profile | no |
+| [`--archive-output`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L206) | Automatically archive the output release directories as bundled archive files, such as zip files. | no |
+| [`--should-build-assets`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L210) | This will process all assets in the project according to asset processor settings, and run the asset bundler as well for specified seedlist files. Omit this if you do not want to process any assets. | no |
```

Comment:
Warn that if assets aren’t built the package won’t come with any assets (pak files) and so won’t run